src/components/DisplayRoutes/Article.jsx

```diff
@@ -14,23 +14,29 @@ export default function Article() {
         const promises = [getArticle(article_id), getCommentsByArticleID(article_id)];
         Promise.all(promises)
         .then(([article,comments]) => {
-            setArticleToDisplay(ArticleCard(article));
+            setArticleToDisplay(article);
             handleCommentsData(comments)
```

Comment:
You could put setCommentsToDisplay(comments) in here rather than invoking a new function to handle the comments